### PR TITLE
Deny a claim

### PR DESCRIPTION
**Pull request by Baggerone**

### CODE REVIEW

File: application/actions/claims_test.go [hunk 1]
```diff
@@ -828,3 +828,110 @@ func (as *ActionSuite) Test_ClaimsApprove() {
...
+			wantInBody: []string{api.ErrorClaimStatus.String()},
+		},
+		{
+			name:       "non-admin user",
+			actor:      policyCreator,
+			oldClaim:   review1Claim,
+			wantStatus: http.StatusNotFound,
+		},
+		{
+			name:       "review1 to review3",
```
Comment (jason-jackson) [inline application/actions/claims_test.go:876]: Test names are off

File: application/actions/claims_test.go [hunk 2]
```diff
@@ -828,3 +828,110 @@ func (as *ActionSuite) Test_ClaimsApprove() {
...
+			wantStatus: http.StatusOK,
+			wantInBody: []string{
+				`"event_description":"` + review1Claim.EventDescription,
+				`"status":"` + string(api.ClaimStatusDenied),
+				`"review_date":"` + time.Now().UTC().Format(domain.DateFormat),
+				`"reviewer_id":"` + appAdmin.ID.String(),
+			},
+		},
+		{
+			name:       "review2 to review3",
```
Comment (jason-jackson) [inline application/actions/claims_test.go:888]: Test names are off

File: application/actions/claims_test.go [hunk 3]
```diff
@@ -828,3 +828,110 @@ func (as *ActionSuite) Test_ClaimsApprove() {
...
+			wantStatus: http.StatusOK,
+			wantInBody: []string{
+				`"event_description":"` + review2Claim.EventDescription,
+				`"status":"` + string(api.ClaimStatusDenied),
+				`"review_date":"` + time.Now().UTC().Format(domain.DateFormat),
+				`"reviewer_id":"` + appAdmin.ID.String(),
+			},
+		},
+		{
+			name:       "review3 to approved",
```
Comment (jason-jackson) [inline application/actions/claims_test.go:900]: Test names are off

File: application/actions/claims_test.go (hunk 1 above)
Comment (Schparky) [inline application/actions/claims_test.go:876]:
```suggestion
			name:       "review1 to denied",
```

File: application/actions/claims_test.go (hunk 2 above)
Comment (Schparky) [inline application/actions/claims_test.go:888]:
```suggestion
			name:       "review2 to denied",
```

File: application/actions/claims_test.go (hunk 3 above)
Comment (Schparky) [inline application/actions/claims_test.go:900]:
```suggestion
			name:       "review3 to denied",
```

**Review by jason-jackson — APPROVED** [review]

File: application/actions/claims_test.go (hunk 1 above)
Comment (Baggerone) [reply application/actions/claims_test.go:876]: :+1:

**Review by Schparky — APPROVED** [review]

File: application/actions/claims_test.go (hunk 1 above)
Comment (Baggerone) [reply application/actions/claims_test.go:876]: :+1:

**Pull request merged**